utils/timeutils/time.go

```diff
@@ -46,3 +47,19 @@ func (t *Time) UnmarshalYAML(unmarshal func(interface{}) error) error {
 
 	return t.UnmarshalJSON([]byte(b))
 }
+
+func GetTickAtTime(timestamp string) int64 {
+	startTime, err := time.Parse(time.RFC3339, config.DefaultConfig.Game.StartDate.Time.String())
```

Comment:
Using the DefaultConfig might lead to issues when we eventually change the game values in our config. If we'd add this function to the api/game package, we'd get access to the current config.

Reply:
That already felt a bit fishy to me when I had to export the DefaultConfig in order to access it, thank you for pointing that out! ówó